src/dune/dune_file.ml

```diff
@@ -630,27 +630,47 @@ module Public_lib = struct
 
   let name t = snd t.name
 
-  let make project ((_, s) as loc_name) =
+  let package t = t.package
+
+  let make ?allow_deprecated_names project ((_, s) as loc_name) =
```

Comment:
We have been using the `?(allow_deprecated_names=false)` form in Dune so far, let's stick to that for consistency.

Reply:
Fixed.